src/main/resources/localstack/s3/keyset_keys/keyset_keys.json

```diff
@@ -1 +1,51 @@
-[]
+[
+  {
+    "id" : 1,
+    "created" : 1609459200,
+    "activates" : 1609459200,
+    "expires" : 1893456000,
+    "secret" : "vPdQIoN1nTwXIQoH4/GcHOAct7ymyrGUKXNQhgmq+E8=",
+    "keyset_id" : 1
+  },
+  {
+    "id" : 2,
+    "created" : 1609459200,
+    "activates" : 1609459200,
+    "expires" : 1893456000,
+    "secret" : "c2yMYeezHfDvZ6haknCFaj0t+eAEJp/C/uh+fUAm1fo=",
+    "keyset_id": 2
+  },
+  {
+    "id" : 3,
+    "created" : 1609459200,
+    "activates" : 1609459200,
+    "expires" : 1893456000,
+    "secret" : "RgyxkP4yP1gYhCINq7O9PxM6jX+etPqSQluZxjB1aG8=",
+    "keyset_id": 3
+  },
+  {
+    "id" : 4,
+    "created" : 1609459200,
+    "activates" : 1609459200,
+    "expires" : 1893456000,
+    "secret" : "bVZZOwCuO0fUf2P6CrwaY23erMErVQiMkGgpy2OMSxc=",
+    "keyset_id": 4
+  },
+  {
+    "id" : 5,
+    "created" : 1609459200,
+    "activates" : 1609459200,
+    "expires" : 1893456000,
+    "secret" : "zpxOafFdffTpeYFvaWnJlaLzJNzk+6qi77U2Umh7QJ4=",
+    "keyset_id": 5
```

Comment:
the keysets.json under localstack shows there's no keyset with id 5 and 6 like below so this caused NullPointerException problem in com.uid2.admin.util.PrivateSiteUtil#getKeysetKeys when

globalKeysets.get(keysetKey.getKeysetId()) tries to globalKeysets.get an non-existing keysetId.

Easiest is to reassign keyset_id above to 821 and the below to something else

Reply:
I have tested locally. Had keysets 1-6 before, I must have overwritten this at some point. I have updated the localstack now

Reply:
similar to my other comment i would suggest to not use same values for keyset id and site id in the localstack json files. and i think we need some more realistic allow_lists values as currently all of the site ids in all allow_lists are invalid site ids (e.g. 2,3,4)?

Reply:
updated to make the lists more realistic and ids more distinctive